.github/workflows/trigger_release.yml

```diff
@@ -50,6 +50,9 @@ jobs:
       - name: tune linux network
         run: sudo ethtool -K eth0 tx off rx off
 
+      # https://github.blog/2023-04-19-introducing-npm-package-provenance/
+      - run: npm i -g npm@9
```

Comment:
Does this need to be bumped in the other workflow as well where publish is actually run?

Reply:
@ijjk Oh yeah I meant to do it in the job that runs `npm publish`. 

Fixed in [0310506](https://github.com/vercel/next.js/pull/48693/commits/0310506b6176316f7935475f28f6d23f376b623d)